ocpi-toolkit-2.2.1/src/main/kotlin/com/izivia/ocpi/toolkit/modules/sessions/services/SessionsValidators.kt

```diff
@@ -49,7 +50,7 @@ fun CdrTokenPartial.validate(): CdrTokenPartial = validate(this) {
 }
 
 fun ChargingPeriodPartial.validate(): ChargingPeriodPartial = validate(this) {
-    validate(ChargingPeriodPartial::startDateTime).isGreaterThan(Instant.now())
+    // startDateTime nothing to validate
```

Comment:
The same as with startDateTime: maybe we need to validate that the date is the current time or in the past.